felles/src/main/kotlin/no/nav/familie/kontrakter/felles/Fødselsnummer.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.familie.kontrakter.felles
+
+import java.time.LocalDate
+
+class Fødselsnummer(val verdi: String) {
```

Comment:
(vet at du bare lagt tilbake, men kanskje kan være fint å endre denne til data class?)
```suggestion
data class Fødselsnummer(val verdi: String) {
```

Reply:
Jeg valgte å fjerne data class, fordi jeg ikke så noen hensikt i å støtte .copy, og siden vi aldri skal skrive ut fødselsnummer ville jeg heller ikke ha en toString implemert. Men det er kanskje tryggere å ovveride toString enn å risikere at noen andre kommer senere å legger til data.